envwarden

```diff
@@ -73,4 +87,4 @@ while read -r key; do
         quoted_value="$(echo "$value" | sed "s/'/'\"'\"'/g")"
         echo export \'$quoted_key\'=\'$quoted_value\'
     fi
-done < <(bw list items --search "$SEARCH" |jq -r '.[].fields[] | select(.name != null) | select(.value != null) | .name, .value')
+done < <(bw list items --search "$SEARCH" |jq -r '.[].fields[]? | select(.name != null) | select(.value != null) | .name, .value')
```

Comment:
do you know what happens if you use this jq command with an older version of jq? Would it show an error? If so, then maybe it's good enough?

Reply:
> For example, if `jq` isn't installed, I think bash would anyway spit out a `command not found` error, 
>so that's good enough that I don't think it's worth checking.
> 
> What do you think?

True, the missing "jq" error is informative enough.

line 90: jq: command not found